graphql/schema.graphql

```diff
@@ -1081,21 +1091,44 @@ type GroupedProjects {
 }
 
 type ProjectSettings {
-  githubWebhooksEnabled: Boolean!
+  gitHubWebhooksEnabled: Boolean!
```

Comment:
This change makes this a breaking change. As long as it isn't being used in production it should be fine.

Reply:
Not being used in production so should be fine 👍